stack.yaml

```diff
@@ -90,9 +91,9 @@ extra-deps:
     commit: 5e0b8bc8c7862be12da6989440f8644ba7c1e1cf
     subdirs:
       - binary
+      - cardano-crypto-praos
       - binary/test
       - cardano-crypto-class
-      - cardano-crypto-praos
```

Comment:
It helps to avoid semanitcally null changes like this. And before you changed them, the were actually sorted alphabetically and now they are not (OCD alert!!!).

Reply:
I actually sorted them as they appear in Cabal but then Stack got changed a couple of times and I didn't bother to fix that.